helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1444,21 +1323,17 @@ operator:
       issuerAnnotations:
         "helm.sh/hook": post-install,post-upgrade
         "helm.sh/hook-weight": "1"
-
 # The cert-manager is a CNCF application deployed as a subchart and used for supporting operators that require TLS certificates.
 # Full list of Helm value configurations: https://artifacthub.io/packages/helm/cert-manager/cert-manager?modal=values
 certmanager:
   enabled: false
   installCRDs: true
-
-
```

Comment:
@jvoravong can we change the automation to avoid the reformatting? I think it reduces readability of the value.yaml

Reply:
Yeah I have a feature branch to update/fix this workflow that I will open on Monday
- Update only the line that contains the image tag value
- Update the git config so the git history shows the author is the bot
- Include the proper version (current/latest) values in the PR 
- Add/update a changelog entry